examples/data_labeler/data_labeler_predict_train.ipynb

```diff
@@ -0,0 +1,370 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "framed-portfolio",
+   "metadata": {},
+   "source": [
+    "# Complex Data Type Detection using Data Labeler component of Data Profiler"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "nutritional-aluminum",
+   "metadata": {},
+   "source": [
+    "In this example, we utilize the data labeler component of the Data Profiler to detect the sensitive information for both structured and unstructured data. Data Profiler can be transfer learned with the new datasets to get better results on some specific domains.\n",
```

Comment:
Data Profiler can be transfer learned with the new datasets to get better results on some specific domains.
fixes to:
The Data Profiler can utilize transfer learning with a new datasets to improve domain specific results.

Reply:
changed